manifests/init.pp

```diff
@@ -276,6 +276,7 @@
   # environment variables
   #..................................................................................
   $config_path                = undef,
+  $file_mode                  = '0600',
```

Comment:
I would not put this new parameter under the category "environment variables", which is related to Tomcat's *setenv*, just for the sake of clarity. Maybe under `tomcat_user` and `tomcat_group`?